engine/include/cubos/engine/plugins/tools/entity_selector.hpp

```diff
@@ -0,0 +1,20 @@
+#ifndef CUBOS_ENGINE_PLUGINS_ENTITY_SELECTOR_HPP
+#define CUBOS_ENGINE_PLUGINS_ENTITY_SELECTOR_HPP
+
+#include <cubos/engine/cubos.hpp>
+#include <cubos/core/ecs/entity_manager.hpp>
```

Comment:
Add a newline between the includes, for consistency.
We usually separate the 'engine' includes from the 'core' includes.